Gemfile

```diff
@@ -14,6 +14,7 @@ group :benchmark, :test do
   gem 'memory_profiler'
   gem 'terminal-table'
   gem "lru_redux"
+  gem "webrick"
```

Comment:
Do we need the `webrick` gem to be  in the `benchmark/test` group?

Reply:
Actually, I've moved `webrick` to the `development` group. I believe it fits better there, as neither `test` nor `benchmark` are so representative of its purpose.